### ISSUE TITLE
Add database error handling to PHP,Javascript

### ISSUE THREAD

**Issue by brentfraser** [nickfox/GpsTracker#64]
A common problem is getting the user's permissions for the database right.  It would be nice to pass the database error from the PHP to the JavaScript for the purposes of getting the system running (maybe a "debug" mode?).

**Comment by nickfox** [comment]
Yes, I agree. That would be very helpful. Over the past 10 years, the number one issue is almost always permission problems.

**Comment by brentfraser** [comment]
And some kind of "ping" from the phone clients would be good.  Could be as simple as issue a plain request (without the lat/lon) to the URL and check that it responded with a "-1".  That would confirm users typed in the right URL.
